packages/wells/src/client/model/Well.ts

```diff
@@ -1,5 +1,8 @@
 import { WellHeadLocation } from './WellHeadLocation';
 
+// Customizable function that takes in CogniteClient and args, and return a promise of a well
+export type SearchWell = (args: any) => Promise<Well[]>;
+
```

Comment:
All custom methods will take in `any` as argument and have to return a `Promise<Well[]>`

Reply:
I don't know JS, so I'm way out of my comfort zone here and may be saying something very wrong, but maybe we should have one that returns multiple wells (`Promise<Well[]>)` and another one that returns a single well (`Promise<Well>`)?